content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,221 @@
+
+```mermaid
+graph TD
+    A[Primary data at one concentration]-->B{Number of readouts per molecules?}
+    B-->|one|V{Which unit?}
+    V-->|percentage|D
+    V-->|value|F
+    B-->|multi|C[valid ones]
+    C-->|experiments|D{if triplicates}
+    D-->|aggregating or filtering|E[validated ones]
+    E--> |calculating|G[One average readout value <br> per concentration]
+    G-->F[CC/AC/IC50/IC90 curves]
+```
+
+### BioActivity data
+
+In the chemical biology domain (CBD), bioactivity data refers to datasets coming from high- to low-throughput biochemical and cellular screenings, hit confirmatory assay and profiling assays. Such datasets usually contain IC/AC/CC/EC50 (where 50 can be any number referring to the percent effect of the readout results). 
+
+Public CBD datasets are released in archives such as [ChEMBL](https://www.ebi.ac.uk/chembl/), [PubChem](https://pubchem.ncbi.nlm.nih.gov/), and [ECBD](https://ecbd.eu/).  They include general project- and sample-level information (like assay summaries or protocols), which will be briefly covered in this recipe but will also be covered in more depth elsewhere.
+
+A specific of this guideline is that they are referring mainly to ChEMBL formatting for uploading, so the covered suggested ontologies are intrinsically linked to “ChEMBL supported” ontologies ([https://www.ebi.ac.uk/ols/index](https://www.ebi.ac.uk/ols/index))
+
+
+#### Bioactivity data formats and standards
+
+
+<table>
+  <tr>
+   <td><strong>Data format</strong>
+   </td>
+   <td><strong>Terminology</strong>
+   </td>
+   <td><strong>Data models</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Chemistry (SDF)
+   </td>
+   <td>Structure ID
+   </td>
+   <td>
+<ul>
+
+<li>SDF
+
+<li>SMILE
+
+<li>InChI
+
+<li>CID
+</li>
+</ul>
+   </td>
+  </tr>
+  <tr>
+   <td>Target 
+   </td>
+   <td>Protein/GENE ID
+   </td>
+   <td>PN_ or SwissProt ID
+   </td>
+  </tr>
+  <tr>
+   <td>Assay
+   </td>
+   <td>Typology
+   </td>
+   <td>Binding, FRET, SPR, Inhibition, phenotypic cellular
+   </td>
+  </tr>
+  <tr>
+   <td>Result Type
+   </td>
+   <td>Potency/Tox 
+   </td>
+   <td>CC50/IC50/EC50/%
+   </td>
+  </tr>
+  <tr>
+   <td>Unit
+   </td>
+   <td>Result unit
+   </td>
+   <td>Concentration/ratio/SI
+   </td>
+  </tr>
+  <tr>
+   <td>Image
+   </td>
+   <td>
+   </td>
+   <td>
+<ul>
+
+<li>OMETIFF
+
+<li>Matrix Format-Zarr
+</li>
+</ul>
+   </td>
+  </tr>
+</table>
+
+
+
+#### Minimum metadata 
+![](https://i.imgur.com/aU2KYV1.png)
+
+
+Minimum metadata set constitutes metadata items that always need to be supplied with any experimental data. For ChEMBL submission, molecular structures and assay description as depicted in the scheme above are suggested as essential metadata. This is basically a subset of the following [schema](https://www.ebi.ac.uk/chembl/db_schema). In case mutated cell lines and/or mutated target proteins have been used in the assay, desirable metadata should be added in the proper group.
```

Comment:
```suggestion
**Minimum metadata** set constitutes metadata items that should ideally be systematically supplied with any experimental data. QUESTION/SUGGESTION: refer to MIABE (Minimum information about a BioActive Entity: https://fairsharing.org/FAIRsharing.dt7hn8). 
For ChEMBL submission, molecular structures and assay description as depicted in the scheme above are suggested as essential metadata. This is basically a subset of the following [schema](https://www.ebi.ac.uk/chembl/db_schema).
In case mutated cell lines and/or mutated target proteins have been used in the assay, additional desirable metadata should be added in the proper group. [QUESTION: is the chembl schema aligned with MIABE and if so how)
```